index.js

```diff
@@ -15,7 +15,6 @@ module.exports = {
     const skygear = new skygearCloud.CloudCodeContainer();
     skygear.endPoint = skygearCloud.settings.skygearEndpoint+'/';
     skygear.apiKey = skygearCloud.settings.masterKey;
-    skygear.pubsub.reconfigure();
```

Comment:
It seems that this `reconfigure` is required here, and this is missing in v1 skygear pubsub container. So I would fix this and you can keep this line here.